README.md

```diff
@@ -0,0 +1,33 @@
+# Sern CLI
+
+Our CLI allows you to setup and manage Discord bot project without writing a single line of code!
+
+😁 **User Friendly** <br>
+💦 **Easy-to-use** <br>
+🌱 **Efficient** <br>
+💪 **Powerful** <br>
+
+## Installation
+
```

Comment:
You could add `NPM`, `Yarn` and `PNPM` titles for the installation part. Would make it a bit more user friendly I believe.